app/src/main/resources/application.yml

```diff
@@ -97,6 +110,21 @@ spring:
   http:
     encoding:
       force: true
+
+  datasource:
+    url: ${POSTGRES_URL}
+    username: ${POSTGRES_USER}
+    password: ${POSTGRES_PASSWORD}
+    hikari:
+      jdbc-url: ${POSTGRES_URL}
+      username: ${POSTGRES_USER}
+      password: ${POSTGRES_PASSWORD}
+      schema: ${POSTGRES_SCHEMA}
+  jpa:
+    generate-ddl: false
+    hibernate:
+      ddl-auto: none
+
```

Comment:
This block came from `application-compose.yml` (which is now renamed to `application-local.yml`).
`application-compose.yml` was used for prod deployments, which it shouldn't be.